File: BEE 1010.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
// var lines = input.split('\n');
var sep = [' ', '\n'];
// console.log(separators.join('|'));
var line = input.split(new RegExp(sep.join('|'), 'g'));

var p1 = parseInt(line.shift());
var u1 = parseInt(line.shift());
var price1 = parseFloat(line.shift());

var p2 = parseInt(line.shift());
var u2 = parseInt(line.shift());
var price2 = parseFloat(line.shift());

console.log("VALOR A PAGAR: R$", ((u1*price1)+(u2*price2)).toFixed(2));

File: BEE 1010.py
line1 = input().split(" ")
p1 = int(line1[0])
u1 = int(line1[1])
price1 = float(line1[2])

line2 = input().split(" ")
p2 = int(line2[0])
u2 = int(line2[1])
price2 = float(line2[2])

print("VALOR A PAGAR: R$ {:.2f}".format((u1*price1)+(u2*price2)))